source/ShipyardPanel.cpp

```diff
@@ -18,9 +18,9 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "text/alignment.hpp"
 #include "comparators/BySeriesAndIndex.h"
 #include "ClickZone.h"
-#include "Color.h"
 #include "Dialog.h"
 #include "text/DisplayText.h"
+#include "FillShader.h"
```

Comment:
already included with ShopPanel
```suggestion
```

Reply:
There must be something more to it then that I'm not fully grokking.

![image](https://github.com/user-attachments/assets/fad55053-86a6-48dd-8c58-7e518381ff46)

Shown above, I fiddled with line 408, changing FillShader to ShopPanel when all the FillShader's turned red upon implementing this fix.

CLion suggests I add it back:
![image](https://github.com/user-attachments/assets/ba1afe3e-f0fb-426c-a992-7b06035c6ca6)

Please help me understand.

Reply:
You're right. Sorry, I looked at ShopPanel.cpp, not ShopPanel.h